posts/simprocs-intro.md

```diff
@@ -0,0 +1,550 @@
+---
+author: 'Yaël Dillies, Paul Lezeau'
+category: 'Metaprogramming'
+date: 2025-03-29 12:33:00 UTC+00:00
+description: 'Brief introduction to simprocs and what they are useful for'
+has_math: true
+link: ''
+slug: what-is-a-simproc
+tags: 'simp, simproc, meta'
+title: What is a simproc?
+type: text
+---
+
+Lean core recently added support for custom simplification procedures, aka *simprocs*.
```

Comment:
```suggestion
Lean core recently added support for custom simplification procedures, aka *simprocs*.
```
Is "recent" true? I thought these have been around for quite a while, at least a year.

Reply:
You are right, they are a year and a half old by now. The PR that introduced them was https://github.com/leanprover/lean4/pull/3124